src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -97,43 +105,14 @@ internal class LaunchSettingsProvider : OnceInitializedOnceDisposed, ILaunchSett
 
         private SequencialTaskExecutor _sequentialTaskQueue = new SequencialTaskExecutor();
 
-        /// <summary>
-        /// Returns the full path to the launch settings file
-        /// </summary>
-        private string _launchSettingsFile;
+        [Obsolete("Use GetLaunchSettingsFileAsync instead.")]
```

Comment:
Did you add GetLaunchSettingsFileAsync to the ILaunchProviderInterface? I don't see that change.